### DISCUSSION TITLE
There is no tracked object with id ??

### DISCUSSION THREAD

**Discussion by djjohnson-net** [mrpmorris/Fluxor#395] [Q&A]
![image](https://user-images.githubusercontent.com/31104515/211101234-b714da4f-93df-44bb-9833-c0f04d09f34e.png)

Is there an easy way to track down what is causing this error?

**Comment by mrpmorris** [comment]
Do you create DotNetObjectReference instances in your code?

**Reply by djjohnson-net** [reply]
Not exactly sure what you mean by "Do you create DotNetObjectReference instances in your code?"

For the most part I do use @ref on my controls. If that is what you are referring too?

**Comment by mrpmorris** [comment]
Did you have Redux Dev Tools enabled? If not, then I don't have any code that calls from JS into dotnet, so the problem must lie elsewhere.

**Reply by djjohnson-net** [reply]
Ok. Thanks. I'm installing Redux Dev Tools.

**Reply by mrpmorris** [reply]
I wasn't recommending you use it to find the case, I was saying that's the only time I call from JS into dotnet.

**Comment by mrpmorris** [comment]
Did you get to the bottom of this?

**Reply by djjohnson-net** [reply]
No. I have been doing some research and can't find anything that is related. Do you have any suggestions? The issue is that I don't know how to figure out what is causing the issue. I was thinking if I can figure out what the object is causing the error a light come on?

**Comment by mrpmorris** [comment]
Are you able to trigger the problem on demand? If so, what user actions do you have to perform?
